spec/std/random/system_spec.cr

```diff
@@ -0,0 +1,10 @@
+require "spec"
+require "random/system"
+
+describe "Random::System" do
+  rng = Random::System.new
+
+  it "returns random numbers from the secure system source" do
+    typeof(rng.next_u).should eq(UInt32)
```

Comment:
Wrong assumption on my side. I was lured by arc4random, libsodium's sysrandom, our ISAAC and MT19937 implementations all returning an UInt32. I see the ISAAC64 algorithm returns an UInt64, so as long as `Random` handles any size, I'll remove the spec limitation.